src/server/game/Spells/Spell.h

```diff
@@ -36,6 +36,8 @@ class ByteBuffer;
 
 #define SPELL_CHANNEL_UPDATE_INTERVAL (1 * IN_MILLISECONDS)
 
+#define SPELL_INTERRUPT_NONPLAYER 32747
```

Comment:
`#define` is bad and you should feel bad.

`enum` or `static const uint32` please.